src/cli/command-package.ts

```diff
@@ -22,14 +23,27 @@ import {readJson} from '../manifest-converter';
 
 export default async function run(options: CliOptions) {
   // Ok, we're updating a package in a directory not under our control.
-  // We need to be sure it's safe. In a future PR let's check with git, but
-  // for now, we'll ask the user to pass in a --force flag.
-  if (!options.force) {
-    console.error(
-        `When running modulizer on an existing directory, ` +
-        `be sure that all changes are checked into source control. ` +
-        `Run with --force once you've verified.`);
-    process.exit(1);
+  // We need to be sure it's safe.
+  try {
+    await new Promise((resolve, reject) => {
```

Comment:
That's funny, I needed this for the fixture tests as well. I ended up implementing a wrapper interface that's been really useful in polymer-workspaces: https://github.com/Polymer/polymer-modulizer/blob/6791dd4ba7ee6960a021d193219879cfa8675d21/src/test/util/exec.ts

Want to bring that file in to your PR and then use it here?

Reply:
Thanks, this definitely cleaned up the implementation. :+1: Should I move it to `src/util.ts` instead of `src/test/util/exec.ts`